### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by olix0r**
Updates all dependencies, but especially the proxy-api, which had a
branch dependency.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+publish = false
 
 [dependencies]
-tokio = { version = "0.2", features = ["full", "parking_lot"] }
-tonic = "0.1.1"
-linkerd2-proxy-api-tonic = { git = "https://github.com/linkerd/linkerd2-proxy-api", branch = "eliza/tonic"}
+tokio = { version = "0.2", features = ["macros", "rt-threaded", "sync"] }
+tonic = "0.2.1"
+linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", features = ["transport"] }
```
Comment (hawkw) [inline Cargo.toml:11]: Should this be pinned to a SHA?

**Review by hawkw — APPROVED** [review]
lgtm!

File: Cargo.toml (hunk 1 above)
Comment (olix0r) [reply Cargo.toml:11]: i think it's okay for now, since the sha is pinned in the lockfile. we'll tag a release soon.

**Review by kleimkuhler — APPROVED** [review]
👍

**Pull request merged**